UnityProject/Assets/Scripts/Player/PlayerEffectsManager.cs

```diff
@@ -22,6 +22,7 @@ private void Awake()
 
     private void Update()
     {
+	    if (PlayerManager.PlayerScript?.PlayerSync == null ) return;
```

Comment:
```suggestion
	    if (PlayerManager.PlayerScript.OrNull()?.PlayerSync == null ) return;
```
PlayerScript is a ManagedNetworkBehaviour which I assume inherits from UnityEngine.Object